src/components/common/layout/Section.js

```diff
@@ -32,7 +32,10 @@ const styles = StyleSheet.create({
   section: {
     backgroundColor: '#fff',
     borderRadius: normalize(5),
-    padding: normalize(8)
+    padding: normalize(8),
+    paddingTop: normalize(15),
```

Comment:
this paddingTop, paddingBotton and marginBotton should be removed I think.

Reply:
Done